server/src/routes/userRoutes.js

```diff
@@ -176,4 +177,6 @@ router.get('/scunt-judge-users', UserController.getScuntJudgeUsers);
  */
 router.delete('/:id', checkLoggedIn, hasAuthScopes(['accounts:delete']), UserController.deleteUser);
 
+router.put('/user-exist', checkLoggedIn, checkUserType('frosh'), UserController.userExist);
```

Comment:
Please add a swagger jsdoc comment - see the route directly above this one for how those look.

Reply:
Addressed here: https://github.com/UofT-Frosh-Orientation/orientation-website/pull/573/commits/b7c05fdf2e97c28b83faef0d19907527139952e5#diff-924ed6689291e383d4e6b5bae4221ee28612e97520455e5d9594f1c3d54addd3R180

Reply:
Thanks!